programs/create-genesis/ee_genesis/genesis_ee_builder.cpp

```diff
@@ -31,15 +31,18 @@ genesis_ee_builder::genesis_ee_builder(const genesis_info& info, const export_in
 genesis_ee_builder::~genesis_ee_builder() {
 }
 
-golos_dump_header genesis_ee_builder::read_header(bfs::ifstream& in) {
+golos_dump_header genesis_ee_builder::read_header(bfs::ifstream& in, const bfs::path& file) {
```

Comment:
`golos_dump_header` is not used, when `in` is opened in this function.

Looks better, if function will return `bfs::ifstream`.

Reply:
@afalaleev yes, can be renamed to `open_dump()` with returning `bfs::ifstream`